### PR TITLE
Update node js packages required by Gerrit

### PR DESCRIPTION
**Pull request by davido**
Fixes: #717.

### CODE REVIEW

File: buildkite/docker/Dockerfile [hunk 1]
```diff
@@ -99,7 +99,15 @@ RUN apt-get -qqy update && \
 RUN curl -L https://deb.nodesource.com/setup_8.x | bash - && \
```
Comment (lucamilanesio) [inline buildkite/docker/Dockerfile]: NodeJS should be upgraded to 10.x also, see the one used in [Gerrit-CI](https://gerrit.googlesource.com/gerrit-ci-scripts/+/refs/heads/master/jenkins-docker/slave-node-wct/Makefile)

Comment (davido) [reply buildkite/docker/Dockerfile]: Indeed, NodeJs is there as well for Debian: https://deb.nodesource.com/setup_10.x.

**Comment by davido** [comment]
Testing it now, and it was not successful:

```
> node scripts/build-browser-scripts

/usr/lib/node_modules/polymer-cli/node_modules/mkdirp/index.js:90
                    throw err0;
                    ^

Error: EACCES: permission denied, mkdir '/usr/lib/node_modules/polymer-cli/node_modules/wd/build'
    at Object.mkdirSync (fs.js:757:3)
    at sync (/usr/lib/node_modules/polymer-cli/node_modules/mkdirp/index.js:71:13)
    at Object.<anonymous> (/usr/lib/node_modules/polymer-cli/node_modules/wd/scripts/build-browser-scripts.js:6:1)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
npm WARN eslint-config-google@0.12.0 requires a peer of eslint@>=5.4.0 but none is installed. You must install peer dependencies yourself.
npm WARN sinon-chai@2.14.0 requires a peer of chai@>=1.9.2 <5 but none is installed. You must install peer dependencies yourself.
npm WARN tsickle@0.23.6 requires a peer of typescript@2.3.4 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/polymer-cli/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wd@1.11.2 install: `node scripts/build-browser-scripts`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the wd@1.11.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-06-12T21_00_48_752Z-debug.log
The command '/bin/sh -c curl -L https://deb.nodesource.com/setup_10.x | bash - &&     apt-get -qqy update &&     apt-get -qqy install nodejs &&     npm install -g         eslint@"=4.19.0"         eslint-config-google@"=0.12.0"         eslint-plugin-html@"=5.0.3"         eslint-plugin-promise@"=4.0.1"         typescript         fried-twinkie@"^0.2.2"         polylint polymer-cli         web-component-tester@6.8.0 &&     rm -rf /var/lib/apt/lists/*' returned a non-zero code: 1
```

**Comment by davido** [comment]
OK, I figured it out, it is `--unsafe-perm` option to npm install is missing. With this option all is fine:

```
+ eslint-plugin-promise@4.0.1
+ fried-twinkie@0.2.2
+ eslint-config-google@0.12.0
+ eslint-plugin-html@5.0.3
+ eslint@4.19.0
+ polylint@2.10.4
+ web-component-tester@6.8.0
+ polymer-cli@1.9.10
+ typescript@3.5.1
added 2540 packages from 1160 contributors in 80.735s
Removing intermediate container e8c36eddca0d
 ---> 953d5529248d
```

**Comment by philwo** [comment]
Thanks a lot! Do you want to update your PR to do the upgrade to Node 10.x, too? I'll merge it then :)

**Comment by davido** [comment]
> Thanks a lot! Do you want to update your PR to do the upgrade to Node 10.x, too? I'll merge it then :)

Thanks! I'm working on it right now! Docker image was built and I re-running the tests with the local built image, just to make sure...

**Comment by davido** [comment]
@philwo JavaScript tests are passing in new docker image:

```
$ bazel test polygerrit-ui/...
INFO: Invocation ID: 91dd0fda-5f11-48b7-a624-75486aea997a
INFO: Analyzed 10 targets (3 packages loaded, 235 targets configured).
INFO: Found 10 test targets...
INFO: Elapsed time: 200.743s, Critical Path: 198.76s
INFO: 21 processes: 1 linux-sandbox, 20 local.
INFO: Build completed successfully, 42 total actions
//polygerrit-ui/app:template_test_admin                                  PASSED in 163.3s
//polygerrit-ui/app:template_test_change                                 PASSED in 177.8s
//polygerrit-ui/app:template_test_change-list                            PASSED in 115.3s
//polygerrit-ui/app:template_test_core                                   PASSED in 80.7s
//polygerrit-ui/app:template_test_diff                                   PASSED in 148.8s
//polygerrit-ui/app:template_test_edit                                   PASSED in 85.1s
//polygerrit-ui/app:template_test_gr-app                                 PASSED in 80.0s
//polygerrit-ui/app:template_test_plugins                                PASSED in 68.9s
//polygerrit-ui/app:template_test_settings                               PASSED in 141.1s
//polygerrit-ui/app:template_test_shared                                 PASSED in 197.2s

Executed 10 out of 10 tests: 10 tests pass.
There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which oneINFO: Build completed successfully, 42 total actions
```

File: buildkite/docker/Dockerfile (hunk 1 above)
Comment (davido) [reply buildkite/docker/Dockerfile]: Done.

**Pull request merged**